src/frequenz/sdk/timeseries/logical_meter/_logical_meter.py

```diff
@@ -78,6 +78,8 @@ async def _run_formula(
             try:
                 msg = await formula.apply()
                 await sender.send(msg)
+            except asyncio.CancelledError:
+                break
```

Comment:
We probably want to add an error log (or `exception`), right?

Reply:
done